falcon/util/misc.py

```diff
@@ -353,27 +353,43 @@ def secure_filename(filename: str) -> str:
         'Bold_Digit_1'
         >>> secure_filename('Ångström unit physics.pdf')
         'A_ngstro_m_unit_physics.pdf'
+        >>> secure_filename('Ångström unit physics.pdf', max_length=20)
+        'A_ngstro_m_unit.pdf'
 
     Args:
         filename (str): Arbitrary filename input from the request, such as a
             multipart form filename field.
+        max_length (Optional[int]): Maximum allowed length of the sanitized
+        filename. If exceeded, it will be truncated while preserving the extension.
```

Comment:
add 4 some spaces before filename to correctly format it

Reply:
Fixed